src/genotyper.cc

```diff
@@ -608,6 +894,33 @@ Status genotype_site(const genotyper_config& cfg, MetadataCache& cache, BCFData&
         loss_trackers.push_back(LossTracker(site.pos));
     }
 
+    // Setup format field helpers
```

Comment:
suggest breaking this out into a helper function alongside FormatFieldHelper template